spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -620,7 +628,8 @@ def close_client(self, index=None, client=None, ask_recursive=True):
 
     # ---- For execution and debugging
     def run_script(self, filename, wdir, args, debug, post_mortem,
-                   current_client, clear_variables, console_namespace):
+                   current_client, clear_variables, console_namespace,
+                   focus_to_editor):
```

Comment:
Should this new `focus_to_editor` param be a kwarg with a default value? Seems like this is breaking the running file from the project explorer

Reply:
Good point. I think we should make most of these paramenters kwargs so that it's easier to call `run_script` from other plugins. I'll do that as part of this PR.